vllm/worker/neuron_model_runner.py

```diff
@@ -75,6 +76,15 @@ def __init__(
 
         # Lazy initialization.
         self.model: nn.Module  # initialize after load_model.
+        self.model_config.generation_config = GenerationConfig(
+            max_length=self.scheduler_config.max_model_len,
+            do_sample=True,
+            per_batch_line=True,
+            top_k=[1] * self.scheduler_config.max_num_seqs,
```

Comment:
The default for `sampling_params.top_k` is -1. Is there a way we can get default from SamplingParams instead of introducing another default here?

Reply:
we can change it to -1, but the `top_k`, `top_p`, and `temperature` here don't really take any effect as their value will be override with the values in SamplingParams during the `execute_model` call.

Reply:
also -1 is not support for on-device sampling.